fast_carpenter/__main__.py

```diff
@@ -14,6 +14,7 @@
 atup.EventBuilder = EventBuilder
 atup.build_parallel = build_parallel
 logging.getLogger(__name__).setLevel(logging.INFO)
+from .version import __version__
```

Comment:
To comply with PEP8, this import should be 3 lines up next to the other imports.

Reply:
https://travis-ci.com/FAST-HEP/fast-carpenter/jobs/241954792